app/config/nvalues.py

```diff
@@ -119,6 +121,7 @@
     MIN_STAKE_AMOUNT = 500000 * pow(10, NEWRL_TOKEN_DECIMAL)
     STAKE_PENALTY_RATIO = 10
     STAKE_CT_ADDRESS = 'ct1111111111111111111111111111111111111115'
+    SNAPSHOT_BLOCKS = 100000
```

Comment:
@Prannav-k let's set this to 50k? 500000 blocks

Reply:
done